### ISSUE TITLE
scrollTo does NOT work on named element

### ISSUE THREAD

**Issue by autoaim800** [appium/java-client#250]
Issue:
Can be found/located but Failed to scroll to a text element of xpath=.//UIATableView/UIATableCell/UIAStaticText, which has been named via accessibilityIdentifier.

Preliminary analysis,
The name attribute of the element is no longer filled by it's value, can be inspected via Appium Inspector; hence the implementation of IOSElement.java may be wrong.
```java
public MobileElement scrollTo(String text) {
    return (IOSElement) findElementByIosUIAutomation(".scrollToElementWithPredicate(\"name CONTAINS '" + text + "'\")"); 
}
```
shouldn't it be:
```java
return (IOSElement) findElementByIosUIAutomation(".scrollToElementWithPredicate(\"value CONTAINS '" + text + "'\")");
```

**Comment by TikhomirovSergey** [comment]
@autoaim800 
Please reopen this issue or open a new one if the problem is reproduced still.
Please try to use this template: https://github.com/appium/java-client/blob/master/ISSUE_TEMPLATE.md

**Comment by autoaim800** [comment]
Can not find any option to re-open this issue hence created as new as #342. Thanks for your hard work. Best wishes.
